x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -77,6 +79,9 @@ import type {
 } from '../types';
 import type { ExternalCallback } from '..';
 
+import type { FleetAuthzRouteConfig } from '../routes/security';
+import { getAuthzFromRequest, hasRequiredFleetAuthzPrivilege } from '../routes/security';
```

Comment:
Don't have to change but you can move the type import into the same import path on line 83 as 
```
import { type FleetAutzRouteConfig, getAuthzFromRequest, hasRequiredFleetAuthzPrivilege } ...
```